src/adapters/MeetingsJSONAdapter.js

```diff
@@ -432,7 +450,7 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
    */
   async handleLocalShare(ID) {
     await this.updateMeeting(ID, async (meeting) => {
-      const updatedMeeting = {...meeting};
+      const updatedMeeting = {};
 
       if (meeting.localShare) {
         updatedMeeting.localShare.getTracks()[0].stop();
```

Comment:
`updatedMeeting.localShare` is not defined

maybe rename `updatedMeeting` to updates

Reply:
there's a bug below on line 462 `if (meeting.localShare)`

should be the other way around.

you can move the `if` contents in the `else` above, after the `await`